src/server/index.js

```diff
@@ -13,7 +14,7 @@ app.use(cors());
 app.use(express.json());
 // Tell express to use a URL Encoding middleware
 app.use(express.urlencoded({ extended: true }));
-
+app.use(cookieParser())
```

Comment:
What is the purpose of this?

Reply:
(a cookies package) I tried to use cookies but I ended up using the local storage